src/rule-parse.y

```diff
@@ -274,13 +275,27 @@ hdr_expr:
 value_list:
 		value_list ',' value
 			{ $1->append(new MaskedValue($3)); $$ = $1; }
+	|	value_list ',' ranged_value
+			{
+			int numVals = $3->length();
+			for (int idx = 0; idx < numVals; idx++)
```

Comment:
Style nit: we put a space after/before the parentheses of conditions (I changed it merge commit).